src/test/java/org/gridsuite/explore/server/SpreadsheetConfigTest.java

```diff
@@ -118,32 +100,20 @@ public MockResponse dispatch(RecordedRequest request) {
                             null,
                             null  // We'll set specificMetadata to null here as it's handled separately
                     );
-
-                    List<ElementAttributes> elementAttributesList = Collections.singletonList(elementAttributes);
-
-                    return new MockResponse()
-                            .setResponseCode(200)
-                            .setHeader("Content-Type", "application/json")
-                            .setBody(objectMapper.writeValueAsString(elementAttributesList));
+                    return new MockResponse(200, Headers.of("Content-Type", "application/json"), objectMapper.writeValueAsString(List.of(elementAttributes)));
                 } else if (path.matches("/v1/directories/.*/elements\\?allowNewName=.*") && "POST".equals(request.getMethod())) {
                     ElementAttributes elementAttributes = new ElementAttributes(CONFIG_UUID, CONFIG_NAME, "SPREADSHEET_CONFIG", USER_ID, 0L, null);
-                    return new MockResponse()
-                            .setResponseCode(200)
-                            .setHeader("Content-Type", "application/json")
-                            .setBody(objectMapper.writeValueAsString(elementAttributes));
+                    return new MockResponse(200, Headers.of("Content-Type", "application/json"), objectMapper.writeValueAsString(elementAttributes));
                 } else if (path.matches("/v1/directories/" + PARENT_DIRECTORY_UUID + "/elements")) {
-                    return new MockResponse().setResponseCode(200).addHeader("Content-Type", "application/json; charset=utf-8");
+                    return new MockResponse(200, Headers.of("Content-Type", "application/json; charset=utf-8") /*, body: TODO content ???*/);
```

Comment:
In other services you just put MockResponse(200)

Reply:
Yes, replace-all haven't take that line after.